components/button.tsx

```diff
@@ -1,21 +1,38 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition } from '@fortawesome/free-solid-svg-icons';
```

Comment:
its weird that you have to import the type from this module. How did you find the type you needed?

Reply:
idk, I just assumed it was like python and retraced how imports work in other files